.github/workflows/tests-template.yml

```diff
@@ -0,0 +1,61 @@
+---
+name: Reusable Integration Workflow
+on:
+  workflow_call:
+    inputs:
+      processor:
+        required: true
+        type: string
+      gofailMake:
+        required: false
+        type: string
+        default: "no"
+permissions: read-all
+
+jobs:
+  test-linux:
+    strategy:
+      fail-fast: false
+      matrix:
+        target:
+          - linux-${{ inputs.processor }}-unit-test-1-cpu
+          - linux-${{ inputs.processor }}-unit-test-2-cpu
+          - linux-${{ inputs.processor }}-unit-test-4-cpu
+          - linux-${{ inputs.processor }}-unit-test-4-cpu-race
+    runs-on: actuated-arm64-8cpu-32gb // need help here
```

Comment:
@ahrtr I need help here :/

Reply:
Please refer to 
https://github.com/etcd-io/etcd/blob/649b3659379c780ffa8f44723ac3410e1b922cfd/.github/workflows/robustness-template.yaml#L28C45-L28C45

Reply:
done